consensus/common_test.go

```diff
@@ -199,7 +212,9 @@ func signAddVotes(
 
 func validatePrevote(t *testing.T, cs *State, round int, privVal *validatorStub, blockHash []byte) {
 	prevotes := cs.Votes.Prevotes(round)
-	address := privVal.GetPubKey().Address()
+	pv, err := privVal.GetPubKey()
```

Comment:
pv as private validator?

```suggestion
	pubKey, err := privVal.GetPubKey()
```